File: LABS/LAB_03/src/ki304/vasylkiv/lab3/Address.java
package ki304.vasylkiv.lab3;

/**
 * Class <code>Address</code> implements the address of the person's place of residence
 * @author Rostyslav Vasylkiv
 * @version 1.0
 */
public class Address {
	private String сountry;
	private String city;
    private String street;
    private int houseNumber;

    /**
     * Constructor
     */
    public Address() {
    	сountry = "unknown";
        city = "unknown";
        street = "unknown";
        houseNumber = 0;
    }
    
    /**
     * Constructor
     * 
     * @param newCountry      New country
     * @param newCity         New city
     * @param newStreet       New street
     * @param newHouseNumber  New house number
     */
    public Address(String newCountry, String newCity, String newStreet, int newHouseNumber) {
    	сountry = newCountry;
        city = newCity;
        street = newStreet;
        houseNumber = newHouseNumber;
    }
   
    
    /**
     * The method is used to change the country
     *
     * @param newCountry 
     */
    public void setCountry(String newCountry) {
        сountry = newCountry;
     }
    
    /**
     * The method is used to change the city
     *
     * @param newCity 
     */
    public void setCity(String newCity) {
    	city = newCity;
    }
    
    /**
     * The method is used to change the street
     *
     * @param newStreet 
     */
    public void setStreet(String newStreet) {
    	street = newStreet;
    }
    
    
    /**
     * The method is used to change the house number
     *
     * @param newHouseNumber 
     */
    public void setHouseNumber(int newHouseNumber) {
    	houseNumber = newHouseNumber;
    }
    
    /** 
     * Method returns сountry 
     * @return String
     */
    public String getCountry() {
        return сountry;
     }
    
    /** 
     * Method returns city
     * @return String
     */
    public String getCity() {
       return city;
    }

    /** 
     * Method returns street
     * @return String
     */
    public String getStreet() {
        return street;
    }

    /** 
     * Method returns houseNumber
     * @return int
     */
    public int getHouseNumber() {
        return houseNumber;
    }
 
    /** 
     * Method returns сountry, city, street
     * @return String
     */
    public String getToString() {
        return сountry + ", " + city + ", " + street + " " + houseNumber;
    }
}

File: LABS/LAB_09/Address.py
class Address:
    # Constructor
    def __init__(self, country="unknown", city="unknown", street="unknown", houseNumber=0):
        self.__country = country
        self.__city = city
        self.__street = street
        self.__house_number = houseNumber

    # Method: Sets the country attribute of the Address object.
    def setCountry(self, newCountry):
        self.__country = newCountry

    # Method: Sets the city attribute of the Address object.
    def setCity(self, newCity):
        self.__city = newCity

    # Method: Sets the street attribute of the Address object.
    def setStreet(self, newStreet):
        self.__street = newStreet

    # Method: Sets the house_number attribute of the Address object.
    def setHouseNumber(self, newHouseNumber):
        self.__house_number = newHouseNumber

    # Method: Returns the country attribute of the Address object.
    def getCountry(self):
        return self.__country

    # Method: Returns the city attribute of the Address object.
    def getCity(self):
        return self.__city

    # Method: Returns the street attribute of the Address object.
    def getStreet(self):
        return self.__street

    # Method: Returns the house_number attribute of the Address object.
    def getHouseNumber(self):
        return self.__house_number

    # Method: Returns a formatted string representation of the Address object.
    def toString(self):
        return f"{self.__country}, {self.__city}, {self.__street} {self.__house_number}"
